rllib/examples/learners/ppo_with_torch_lr_schedulers.py

```diff
@@ -0,0 +1,147 @@
+"""Example of how to use PyTorch's learning rate schedulers to design a complex
```

Comment:
Let's add this to BUILD:

search for "# subdirectory: learners/" and add it somewhere under there (alphabetical).